util/src/main/java/tech/pegasys/artemis/util/config/ArtemisConfiguration.java

```diff
@@ -54,6 +54,12 @@ static final Schema createSchema() {
     builder.addString("node.bootnodes", "", "ENR of the bootnode", null);
     builder.addString(
         "validator.validatorsKeyFile", "", "The file to load validator keys from", null);
+    builder.addString(
+        "validator.validatorsKeystoreConfFile",
+        "",
+        "The file containing paths to encrypted keystore files and password files to decrypt them",
+        null);
```

Comment:
Why do we need a config entry that points to another config file that lists the keystores?

Reply:
I'll change it to specify all the keystores files and corresponding password files instead of using another configuration file.